Source/Laser/LaserEnvelope.H

```diff
@@ -0,0 +1,13 @@
+#include <complex>
+#include <iostream>
+#include <complex>
+#include <vector>
+
+
+#include <iostream>
+#include <vector>
+
+class LaserEnvelope {
```

Comment:
In addition, you need a constructor (which would typically read the input script)
```suggestion
class LaserEnvelope {
     public:
           LaserEnvelope::LaserEnvelope();
```

Reply:
Here you might want to read the number of MR levels as in the constructor of the class `HybridPICModel`: https://github.com/ECP-WarpX/WarpX/blob/de8e9622f39bec29369c37907a6d854fc619aacf/Source/FieldSolver/FiniteDifferenceSolver/HybridPICModel/HybridPICModel.H#L32